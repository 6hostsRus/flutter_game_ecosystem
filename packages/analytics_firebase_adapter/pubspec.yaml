
name: analytics_firebase_adapter
description: Firebase Analytics sink for core_services AnalyticsSink.
version: 0.1.0
environment:
  sdk: ">=3.3.0 <4.0.0"
dependencies:
  flutter:
    sdk: flutter
  firebase_core: ^2.32.0
  firebase_analytics: ^10.10.0
  flutter_riverpod: ^2.5.1
  core_services:
    path: ../core_services
dev_dependencies:
  flutter_lints: ^4.0.0
