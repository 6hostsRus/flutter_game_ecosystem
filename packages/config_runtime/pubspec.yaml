name: config_runtime
description: Unified config discovery/merge/validation runtime for all engines.
version: 0.1.0
environment: { sdk: '>=3.3.0 <4.0.0' }
dependencies:
     yaml: ^3.1.2
     path: ^1.9.0
     collection: ^1.18.0
     csv: ^6.0.0
     json_schema2: ^5.1.3
     meta: ^1.12.0
dev_dependencies:
     lints: ^3.0.0
     test: ^1.25.0
flutter:
     assets: [assets/schemas/]
