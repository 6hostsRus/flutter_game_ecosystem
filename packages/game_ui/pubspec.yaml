name: game_ui
description: Reusable UI components for Flutter/Flame games (bottom nav, HUD, store, upgrades).
version: 0.1.0
environment:
  sdk: ">=3.3.0 <4.0.0"
dependencies:
  flutter:
    sdk: flutter
  go_router: ^14.2.0
  flutter_riverpod: ^2.5.1
  collection: ^1.18.0
  intl: ^0.19.0
dev_dependencies:
  flutter_test:
    sdk: flutter
  flutter_lints: ^4.0.0
flutter:
  uses-material-design: true
