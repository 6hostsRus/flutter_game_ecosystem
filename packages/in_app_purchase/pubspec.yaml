name: in_app_purchase
version: 0.0.1
publish_to: 'none'
description: Local stub of in_app_purchase plugin for testing / offline dev.

environment:
     sdk: '>=3.3.0 <4.0.0'
     flutter:

flutter:
     plugin:
          platforms: {}

dependencies:
     flutter:
          sdk: flutter

dev_dependencies:
     flutter_test:
          sdk: flutter
