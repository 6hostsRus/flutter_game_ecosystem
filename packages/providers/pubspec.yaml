name: providers
description: Provider bundles (registry, scene router, analytics streams).
version: 0.1.0
publish_to: 'none'

environment:
     sdk: '>=3.3.0 <4.0.0'

dependencies:
     flutter:
          sdk: flutter
     riverpod: ^2.5.1
     hooks_riverpod: ^2.5.1
     services:
          path: ../services

dev_dependencies:
     flutter_test:
          sdk: flutter
     lints: ^5.0.0
