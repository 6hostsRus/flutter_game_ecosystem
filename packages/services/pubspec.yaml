name: services
description: Core service ports (economy, save/profile, telemetry) with no platform impls.
version: 0.1.0
publish_to: 'none'

environment:
     sdk: '>=3.3.0 <4.0.0'

dependencies:
     flutter:
          sdk: flutter

dev_dependencies:
     flutter_test:
          sdk: flutter
     lints: ^5.0.0
