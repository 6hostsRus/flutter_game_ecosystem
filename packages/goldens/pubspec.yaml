name: goldens
description: Golden capture & deterministic replay utilities.
version: 0.1.0
environment:
     sdk: '>=3.3.0 <4.0.0'

dependencies:
     path: ^1.9.0
     meta: ^1.12.0

dev_dependencies:
     flutter_test:
          sdk: flutter
